hazelcast-hibernate/hazelcast-hibernate4/src/test/java/com/hazelcast/hibernate/local/LocalRegionCacheTest.java

```diff
@@ -94,6 +95,16 @@ public void testThreeArgConstructorRegistersTopicListener() {
         verify(topic).addMessageListener(isNotNull(MessageListener.class));
     }
 
+    @Test
+    public void testTimestampToString() {
+        assertNotNull(new Timestamp().toString());
```

Comment:
Just testing on not null doesn't make a lot of sense to me. It leads to a false sense of security in the sense that the code seems tested, but actually there is not a lot of validation logic. Looks nice for coverage; but not a lot of actual value.